config.py

```diff
@@ -25,6 +25,7 @@
 story_time = 446205631159074818
 
 # List of specific roles
+administrator = 0
```

Comment:
Shouldn't this value be set?

Reply:
It could be useful. Do we currently have a test administrator role on the test server?

Reply:
Can somebody set this?